### PR TITLE
feat: add quorum smoke tests

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Smoke tests should also check the status of quorums.

## What was done?

Add tests to check for:
- formation of the proper quorum types
- presence of chainlocks at the chain tip
- recency of the first quorum of an appropriate type for the target network
- presence of IS locks after creating a transaction

## How Has This Been Tested?

Locally, against `devnet-bintang`

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: test/smoke/quorum.js [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+const { inventory, network, variables } = getNetworkConfig();
+
+const allHosts = inventory.masternodes.hosts.concat(
+  inventory.wallet_nodes.hosts,
+  inventory.miners.hosts,
+  inventory.seed_nodes.hosts,
+);
+
+const quorumCheckTypes = {
```
Comment (shumkov) [inline test/smoke/quorum.js:12]: I guess the list should depend on the network, isn't? Or testnet and devnets have the same set?

Comment (strophy) [reply test/smoke/quorum.js:12]: Yeah, I'm not really sure how to configure this properly. dkgInterval is also the wrong measurement, quorums seem to get older than 24 blocks. How does this work?

Comment (shumkov) [reply test/smoke/quorum.js:12]: You probably need to have a map of which quorum types and the number of quorums should exist for testnet and devnet. Then use https://github.com/dashpay/dashcore-lib/blob/master/lib/deterministicmnlist/QuorumEntry.js#L335 to get params for specific quorum type.

**Comment by strophy** [comment]
This will fail until `getMemPoolEntry` is available in `@dashpay/dashd-rpc`. Or I could commit add a patch in here?

File: test/smoke/quorum.js (hunk 1 above)
Comment (strophy) [reply test/smoke/quorum.js:12]: How should I use this? It looks like it returns useful vars about the number of quorum members, should we add a check for that? Is there a similar function I can use to get the quorum lifetimes, or is it fine to use `dkgInterval * 1.5`?

Comment (shumkov) [reply test/smoke/quorum.js:12]: >  It looks like it returns useful vars about the number of quorum members, should we add a check for that?
I don't think so. If we have quorum in list it's already has min number of valid members.
>  Is there a similar function I can use to get the quorum lifetimes
I don't think so

**Review by shumkov — APPROVED** [review]
👍

**Pull request merged**